test/integration/scheduler/volume_binding_test.go

```diff
@@ -288,7 +288,9 @@ func TestVolumeBindingStress(t *testing.T) {
 
 	// Validate Pods scheduled
 	for _, pod := range pods {
-		if err := waitForPodToSchedule(config.client, pod); err != nil {
+		// Use increased timeout for stress test because there is a higher chance of
+		// PV sync error
+		if err := waitForPodToScheduleWithTimeout(config.client, pod, 60*time.Minute); err != nil {
```

Comment:
Per discussion with @msau42 on slack and 2~hrs non-stop testing locally,  I can confirm the issue is fixed. Thanks!

Emm .. do we really need `60*time.Minute` timeout here? I recalled the overall timeout of integration is 300s.

Reply:
Yeah I ran this test over 1000 times until all the log files filled up my filesystem :-D

I totally meant 60 seconds, not minutes! Thanks for catching